Tools/scripts/build_options.py

```diff
@@ -34,6 +34,7 @@ def __init__(self,
     Feature('AHRS', 'MicroStrain5', 'AP_EXTERNAL_AHRS_MICROSTRAIN5_ENABLED', 'Enable MICROSTRAIN 5-series External AHRS', 0, "AHRS_EXT"),  # noqa: E501
     Feature('AHRS', 'MicroStrain7', 'AP_EXTERNAL_AHRS_MICROSTRAIN7_ENABLED', 'Enable MICROSTRAIN 7-series External AHRS', 0, "AHRS_EXT"),  # noqa: E501
     Feature('AHRS', 'AHRS_EXT_VECTORNAV', 'AP_EXTERNAL_AHRS_VECTORNAV_ENABLED', 'Enable VectorNav External AHRS', 0, "AHRS_EXT"),  # noqa
+    #Feature('AHRS', 'InertialLabs', 'AP_EXTERNAL_AHRS_INERTIAL_LABS_ENABLED', 'Enable InertialLabs External AHRS', 0, "AHRS_EXT"), # disabled because INERTIAL_LABS is the enable flag
```

Comment:
There are 9 instances of `AP_EXTERNAL_AHRS_INERTIAL_LABS_ENABLED` in the code base - I'd like to change them to `AP_EXTERNAL_AHRS_INERTIALLABS_ENABLED` so this line can be uncommented and changed correctly.

We can either detect the InertialLabs class correctly, but not use this `build_option` feature externally, OR, incorrectly detect the feature, but correctly use the `build_options` to generate a file.

What should it be?

Reply:
Of course, this would potentially break peoples custom build servers...

Could rename the class. 

Is there a way to map extracted feature classes to a different define name? I couldn't see one

Reply:
I'm fine with this rename

Reply:
Excellent. I'll push a patch for that shortly.